### PR TITLE
i18n(pt-BR): Add `troubleshooting.md` and update `ui.ts` translations

### PR DESCRIPTION
**Pull request by Yan-Thomas**
#### What kind of changes does this PR include?

- [ ] Minor content fixes (broken links, typos, etc.)
- [ ] New or updated content
- [x] Translated content
- [ ] Changes to the docs site code
- [ ] Something else!

#### Description
Adds PT-BR `troubleshooting.md` page and updates `ui.ts` with translations for the new entries.

### CODE REVIEW

File: src/pages/pt-BR/guides/troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+
+Isto pode ser causado por tentar acessar o objeto `window` ou `document` em tempo de renderização. Por padrão, Astro irá renderizar seu componente [isomorficamente](https://en.wikipedia.org/wiki/Isomorphic_JavaScript), o que significa que ele será executado no servidor, onde o runtime do navegador não está disponível. VOcê pode desabilitar essa etapa de pré-renderização utilizando [a diretiva `client:only`](/pt-BR/reference/directives-reference/#clientonly).
+
+**Solução**: Tente acessar esses objetos após a renderização (ex: [`useEffect()`](https://reactjs.org/docs/hooks-reference.html#useeffect) no React ou [`onMounted()`](https://vuejs.org/api/composition-api-lifecycle.html#onmounted) no Vue e Svelte)
+
+**Status**: Comportamento esperado do Astro, como pretendido.
+
+#### Não é isso?
+
+**Solução**: Verifique a documentação apropriada do seu componente [Astro](/pt-BR/core-concepts/astro-components/) ou de [framework de UI](/pt-BR/core-concepts/framework-components/). Considere abrir um dos templates iniciais do Astro em [astro.new](https://astro.new) e tente solucionar o problema do seu componente em um projeto Astro mínimo.
```
Comment (victor0x16) [inline src/pages/pt-BR/guides/troubleshooting.md:48]:
```suggestion
**Solução**: Verifique a documentação apropriada do seu componente [Astro](/pt-BR/core-concepts/astro-components/) ou a do seu [framework de UI](/pt-BR/core-concepts/framework-components/). Considere abrir um dos templates iniciais do Astro em [astro.new](https://astro.new) e tente solucionar o problema do seu componente em um projeto Astro mínimo.
```

File: src/pages/pt-BR/guides/troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+
+**Solução**: Tente procurar erros em quaisquer componentes que você está importando e renderizando, e certifique-se que estão funcionando corretamente. Considere abrir um dos templates iniciais do Astro em [astro.new](https://astro.new) e tente solucionar o problema do seu componente em um projeto Astro mínimo.
+
+**Status**: Comportamento esperado do Astro, como pretendido.
+
+## Pegadinhas comuns
+
+### Meu componente não está sendo renderizado
+
+Primeiro, verifique-se de que você **importou o componente** no [script do seu componente `.astro`](/pt-BR/core-concepts/astro-components/#o-script-do-componente) ou no [frontmatter do `.md`](/pt-BR/guides/markdown-content/#usar-componentes-no-markdown).
```
Comment (victor0x16) [inline src/pages/pt-BR/guides/troubleshooting.md:65]:
```suggestion
Primeiro, certifique-se de que você **importou o componente** no [script do seu componente `.astro`](/pt-BR/core-concepts/astro-components/#o-script-do-componente) ou no [frontmatter do `.md`](/pt-BR/guides/markdown-content/#usar-componentes-no-markdown).
```

File: src/pages/pt-BR/guides/troubleshooting.md [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+
+Então verifique sua declaração de importação:
+
+- Estaria a sua importação linkando ao lugar errado? (Verifique o caminho da importação.)
```
Comment (victor0x16) [inline src/pages/pt-BR/guides/troubleshooting.md:69]:
```suggestion
- Estaria a sua importação vinculada ao lugar errado? (Verifique o caminho da importação.)
```

File: src/pages/pt-BR/guides/troubleshooting.md [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+
+#### Frontmatter
+
+Uma declaração `console.log()` no frontmatter do Astro irá sempre sair no **terminal** executando a interface de linha de comando do Astro. Isto acontece pois Astro é executado no servidor, e nunca no navegador.
+
+```astro
+---
+const soma = (a, b) => a + b;
+
+// Exemplo: MOstra "4" no terminal da interface de linha de comando
```
Comment (victor0x16) [inline src/pages/pt-BR/guides/troubleshooting.md:136]:
```suggestion
// Exemplo: Mostra "4" no terminal da interface de linha de comando
```

File: src/pages/pt-BR/guides/troubleshooting.md [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+---
+<!-- Exemplo: Todos os três exemplo são equivalentes. -->
+<Debug answer={soma(2, 4)} />
+<Debug {{answer: soma(2, 4)}} />
+<Debug {answer} />
+```
+
+## Precisa de mais?
+
+Venha e papeie conosco no [Discord](https://astro.build/chat) e explique seu problema no canal `#support-threads`. Nós estamos sempre felizes em ajudar!
```
Comment (victor0x16) [inline src/pages/pt-BR/guides/troubleshooting.md:181]:
```suggestion
Venha e fale conosco no [Discord](https://astro.build/chat) e explique seu problema no canal `#support-threads`. Nós estamos sempre felizes em ajudar!
```

**Review by victor0x16 — CHANGES_REQUESTED** [review]
@Yan-Thomas , great work!

Added some suggestions 🚀 !

**Comment by Yan-Thomas** [comment]
Thanks for the review, @victor0x16!!

**Review by victor0x16 — APPROVED** [review]
LGTM!

**Pull request merged**